frontend/src/static/js/components/webstatus-typeahead.ts

```diff
@@ -259,7 +262,7 @@ export class WebstatusTypeahead extends LitElement {
   renderInputField(): TemplateResult {
     return html`
       <sl-input
-        id="inputfield"
+        id=${this.isOverviewSearchbox ? 'inputfield' : ''}
```

Comment:
Any reason to not pass this as a string property and instead have the overview page pass in `inputfield`? Currently, this makes this typeahead component (which we borrow from our ChromeStatus [project](https://github.com/GoogleChrome/chromium-dashboard/blob/main/client-src/elements/chromedash-typeahead.ts)) very dependent on the overview page component.

Otherwise, the else case should be `nothing` and not `''` in this ternary operation. Similar to how [ifDefined](https://github.com/lit/lit/blob/12109c25997ef03180d7eefe05c64e0fb20dd2b0/packages/lit-html/src/directives/if-defined.ts#L6-L15) works

Reply:
`nothing` did not work in this case, since the ID value must be a string - that being said, your suggestion is a good one, and I've changed the implementation accordingly. 🙂